networkit/distance.pyx

```diff
@@ -1253,23 +1259,34 @@ cdef class SPSP(Algorithm):
 	def __dealloc__(self):
 		self._G = None
 
-	def getDistances(self):
-		""" 
-		getDistances()
-		
+	def getDistances(self, bint asarray=False):
```

Comment:
Let's use `None` as default here, that gives us the possibility to switch to NumPy arrays by default in the future.

Reply:
I'm not sure if `asarray` is really a good name here. Would `as_np` or `np_array` be better? Or will most users get what `asarray` is supposed to do anyway?

Reply:
Good point, will update to `None` as default. I borrowed [`asarray`](https://numpy.org/doc/stable/reference/generated/numpy.asarray.html) because it's a commonly used function in numpy to ensure one gets an array. Open to naming suggestions though.

Reply:
Alright, then let's keep `asarray`. We can merge the PR as soon as the `None` default is updated.